app/javascript/components/world-map/WorldMap.js

```diff
@@ -58,6 +59,42 @@ function WorldMap({ zoomToGeographyIso }) {
   // react-select
   const setContentId = selectedOption => dispatch({ type: 'setContentId', payload: selectedOption.value });
   const setContextId = selectedOption => dispatch({ type: 'setContextId', payload: selectedOption.value });
+  const customStyles = {
```

Comment:
I'm totally not a fan of CSS in javascript, but as react-select went this way :man_shrugging:

Reply:
That's why I haven't styled it before, procrastinating things I don't like, but tell no-one :P